public/app/features/dashboard/components/DashboardSettings/JsonEditorSettings.tsx

```diff
@@ -36,7 +39,7 @@ export function JsonEditorSettings({ dashboard, sectionNav }: SettingsPageProps)
         {dashboard.meta.canSave && (
           <div>
             <Button type="submit" onClick={onClick}>
-              Save changes
+              {t('dashboard-settings.json-editor-settings.save-button', 'Save changes')}
```

Comment:
same comment about shortening `json-editor-settings` to just `json-editor` as well